### PR TITLE
adding fork instruction to contributions doc

### PR DESCRIPTION
**Pull request by mdodsworth**
@vraja2 it's not possible to push directly to the optimizely repo: folks need to work from their own fork and create pull requests from there. Adding that step to the contributing doc.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -50,4 +51,4 @@ Your contributions should also include the following header:
 
 ## Contact
 
-If you have questions, please contact developers@optimizely.com.
```
Comment (mdodsworth) [inline CONTRIBUTING.md]: fixed a whitespace issue

**Comment by aliabbasrizvi** [comment]
@mdodsworth I updated some permissions. Can you see if it is any different?

**Comment by mdodsworth** [comment]
still no dice

**Comment by vraja2** [comment]
looks good, thanks!

**Comment by vraja2** [comment]
@mdodsworth we're going to need to hold off temporarily on merging, since some additional legal paperwork needs to be done before accepting contributions. sorry about this!

**Comment by mdodsworth** [comment]
@vraja2 great thanks. BTW, as I can't commit to the optimizely repo, you'll have to merge the pull request (in case there was some confusion there)

**Comment by wangjoshuah** [comment]
@vraja2 are you following up on this?

**Comment by mikeng13** [comment]
Closing this old PR out.

**Pull request closed without merging**